src/Elmish.WPF/BindingInternal.fs

```diff
@@ -410,18 +410,18 @@ module internal BindingData =
   module OneWaySeqLazy =
 
     let mapMinorTypes
```

Comment:
I think this method is clearer with the shorter names. For example, the function `inMapA` is passed arguments `a1` and `a2`.

Reply:
It's to match it with the other names that are used for similar bindings. None of the other bindings use `'a` and `'b`. They all use `'bindingModel` and `'bindingViewModel`. See line 616 in this file for `SubModelSeqUnkeyed` and line 665 for `SubModelSeqKeyed`.

Reply:
Also `SubModelWin` uses the same name, `SubModel` as well but its function is in another file (I move it here in another PR).